### DISCUSSION TITLE
Restore from different database

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#351] [Q&A]
I dont know if here is the right place to make this quiestion:

I have a copier-template on one server, let say I did with database name base1 on S3 backup provider

Now I create a new copier-template project with other databse name, lets say  base2 

I dont know how to restore a backup made it for first copier-template project to this new one database for testing porpouses.

Regards

**Comment by yajo** [comment]
> I dont know if here is the right place to make this quiestion:

No, it wasn't. :laughing: But i moved it [where it belongs](https://github.com/Tecnativa/doodba/discussions?discussions_q=category%3AQ%26A).



> I dont know how to restore a backup made it for first copier-template project to this new one database for testing porpouses.

Your use case requires doing manual stuff. Remember that only `prod.yaml` has defined a `backup` service, so you'd need to:

1. `cd new-project`
1. Configure the backup container with required env variables to make it able to restore `base1` database. Probably that'd mean copying some `.docker/*.env` files from the 1st deployment, or even changing some of them in `common.yaml`.
1. Something similar to `docker-compose -f prod.yaml run --rm backup sh -c 'restore --force && psql -f $SRC/$PGDATABASE.sql'`
1. Then exit that shell, `docker-compose down` and remove all garbage that lets this environment access resources from the other one.
